packages/node/src/__tests__/http-integration.test.ts

```diff
@@ -50,6 +51,9 @@ describe('Analytics Node', () => {
     `
     )
     const body = JSON.parse(httpRes.body)
+    const event = body.batch[0]
+    expect(event.context.library.version).toBe(version)
```

Comment:
Nit: This could be part of line 67 since that's doing a version check.

Reply:
Hmm, adding expressions to the inline-snapshot itself works -- feels weird, but sure!

Reply:
Ooh, actually you can't do that... =(